### ISSUE TITLE
Support for dark mode OFF

### ISSUE THREAD

**Issue by novr** [auramagi/NewIcon#1]
Unreadable when dark mode is disabled.

---
as is: 
text is difficult to read

<img width="123" alt="image" src="https://user-images.githubusercontent.com/47618299/165679915-a0cb912a-adad-456e-8ca4-b14aa28b0295.png">

---
to be: 
text can be read

---
environment: 
Dark mode OFF

Mac OS 12.3.1（21E258）

```
❯ mint run auramagi/NewIcon -h
🌱 Finding latest version of NewIcon
🌱 Running new-icon 0.1.1...
OVERVIEW: macOS icon customization

❯ mint version
Version: 0.17.1

❯ swift --version
swift-driver version: 1.45.2 Apple Swift version 5.6 (swiftlang-5.6.0.323.62 clang-1316.0.20.8)
Target: arm64-apple-macosx12.0

```

---
to reproduce:
```
❯ mint run auramagi/NewIcon text /Applications/Xcode-13.3.1.app 13.3.1 
🌱 Finding latest version of NewIcon
🌱 Running new-icon 0.1.1...
```

---
workaround:
Dark mode ON
Rerun!

**Comment by auramagi** [comment]
@novr 
Thank you for opening this issue!

I fixed this in #2. If you can check that this works, I will merge and make another release. 

`mint run auramagi/NewIcon@feature/issue-1 text /Applications/Xcode-13.3.1.app 13.3.1`

**Comment by auramagi** [comment]
Released as v0.1.2, let me know if you have any other issues.
